tests/integration/conftest.py

```diff
@@ -31,10 +36,93 @@
         COLUMNS.tax_year.name,
         COLUMNS.employer_id.name,
     ],
-    # DATASETS.tax_1040.name: "todo",
+    DATASETS.tax_1040.name: [
+        COLUMNS.simulant_id.name,
+        COLUMNS.tax_year.name,
+    ],
 }
 
 
+@pytest.fixture(scope="session")
```

Comment:
The follow fixtures have been updated to more accurately replicate the actual file structure of our sample data within the pytest framework. Originally, we were creating a tmpdir each time we called a specific dataset. Now using the tmpdir factory, we create a directory within the pytest framework that has the same directory/sub directory for all our datasets.